message agent_server_node
{
	required string server_name = 1;
	required bool is_pipe = 2;
	required string address = 3;
	required int32 port = 4;
	required int32 server_id = 5;
	required bool need_login = 6 [default = false];
};

message agent_servers
{
	repeated agent_server_node servers = 1;
};
